src/models/projectModel.js

```diff
@@ -2,8 +2,8 @@ const mongoose = require('mongoose');
 
 const { Schema } = mongoose;
 
-const Team = mongoose.models.Team;
-const Task = mongoose.models.Task;
+const { Team } = mongoose.models;
+const { Task } = mongoose.models;
```

Comment:
Can you merge these lines please, thanks for the refacto!

```suggestion
const { Team, Task } = mongoose.models;
```